code/datums/wires/robot.dm

```diff
@@ -50,9 +50,6 @@
 				R.show_laws()
 		if(WIRE_LOCKDOWN)
 			R.SetLockdown(!R.lockcharge) // Toggle
-		if(WIRE_RESET_MODULE)
-			if(R.has_module())
-				R.ResetModule()

```

Comment:
At least leave some sort of feedback here, so a Roboticist testing the wires with a multi-tool knows what this one does.